acapi2/resources/acquiadata.py

```diff
@@ -46,7 +46,7 @@ def generate_url_query(uri: str, params: dict) -> str:
         return url
 
     def request(self, uri: str = None, method: str = "GET",
-                data: dict = None, params: dict = None):
+                data: dict = None, params: dict = {}):
```

Comment:
This causes several issues because [default arguments are mutable](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)